src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -191,4 +191,10 @@
      * reads that should have mapped to GC>=50% regions mapped elsewhere.
      */
     public double GC_DROPOUT;
+
+    /** The theoretical HET SNP sensitivity. */
+    public double HET_SNP_SENS;
+
+    /** The Q Score of the theoretical HET SNP sensitivity. */
```

Comment:
Same as above about commenting that this is a phred scale Q score, unless this would be obvious to anyone familiar w/ Picard.